debugger/debugger_parser.mly

```diff
@@ -171,15 +171,16 @@ opt_signed_int64_eol :
 
 longident :
     LIDENT                      { Lident $1 }
-  | module_path DOT LIDENT      { Ldot($1, $3) }
+  | module_path DOT LIDENT      { Ldot(Location.mknoloc $1, Location.mknoloc $3) }
   | OPERATOR                    { Lident $1 }
-  | module_path DOT OPERATOR    { Ldot($1, $3) }
-  | module_path DOT LPAREN OPERATOR RPAREN { Ldot($1, $4) }
+  | module_path DOT OPERATOR    { Ldot(Location.mknoloc $1, Location.mknoloc $3) }
+  | module_path DOT LPAREN OPERATOR RPAREN
+                                { Ldot(Location.mknoloc $1, Location.mknoloc $4) }
```

Comment:
Since nearly all rules are too long, I propose to add a newline after the production, and align from there
```ocaml
    LIDENT
      { Lident $1 }
  | module_path DOT LIDENT
      { Ldot(Location.mknoloc $1, Location.mknoloc $3) }
  | OPERATOR
      { Lident $1 }
  | module_path DOT OPERATOR
      { Ldot(Location.mknoloc $1, Location.mknoloc $3) }
  | module_path DOT LPAREN OPERATOR RPAREN
      { Ldot(Location.mknoloc $1, Location.mknoloc $4) }
 ```